spec/requests/healthcheck_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```

Comment:
This frozen string literal was missed.

Reply:
yeah sorry, noticed this immediately after I sent it over, fixed now